networking/change-block-size.md

```diff
@@ -19,9 +19,67 @@ This how-to guide uses the following {{site.prodname}} features:
 
 ### Concepts
 
+#### About IP pools
+
+The {{site.prodname}} IPAM block sizes for IP pools are:
+
+- Default: /26 (IPv4)
+- IPv4 range: 20-32, inclusive
+- IPv6 range: 116-128, inclusive
+
+You can have **only one default IP pool for per protocol** in your installation manifest. In this example, there is one IP pool for IPv4 (/26), and one IP pool for IPv6 (/116).
+
+```
+apiVersion: operator.tigera.io/v1
+kind: Installation
+metadata:
+  name: default
+  pec:
+   # Configures Calico networking.
+   calicoNetwork:
+     # Note: The ipPools section cannot be modified post-install.
+    ipPools:
+    - blockSize: 26
+      cidr: 10.48.0.0/21
+      encapsulation: IPIP
+      natOutgoing: Enabled
+      nodeSelector: all()
+    - blockSize: 116
```

Comment:
The example actually has two IPv4 pools specified - I'd suggest changing the latter to this:

```
- blockSize: 122
  cidr: 2001::00/64
  encapsulation: None
  natOutgoing: Enabled
  nodeSelector: all()
```

Reply:
Fixed.